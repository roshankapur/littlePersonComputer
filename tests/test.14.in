1018
2019
3118
4215
2019
3017
2119
1020
2021
3120
4112
4001
2020
2121
4001
1121
4300
1
-99999
4
7
7
7
7