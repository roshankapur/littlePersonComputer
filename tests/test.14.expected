***Start LPC Execution***
Contents of 21: 7
***End LPC Execution***
